sensor_msgs/JointState[] sampled_joint_states # Configuration at which to calculate constraints at 
---
constrained_manipulability/PolytopeHyperplanes[] polytope_hyperplanes
